cmd/minishift/cmd/start_preflight.go

```diff
@@ -311,8 +313,8 @@ func checkStorageMounted(driver drivers.Driver) bool {
 // space available.
 func checkStorageUsage(driver drivers.Driver) bool {
 	usedPercentage := getDiskUsage(driver, StorageDisk)
-	fmt.Printf("%s ", usedPercentage)
-	usedPercentage = strings.TrimRight(usedPercentage, "%")
+	fmt.Printf("%s used ", usedPercentage)
+	usedPercentage = stringUtils.GetOnlyNumbers(usedPercentage)
 	usage, err := strconv.ParseInt(usedPercentage, 10, 8)
```

Comment:
I think this can also be written as `strconv.ParseInt(stringUtils.GetOnlyNumbers(usedPercentage), 10, 0)`. because in that case we do not re-use the `userPercentage` variable.

Reply:
I too was thinking of that, but in the interest of not making that line too long (i am using a x230 thinkpad) used the variable again. will change it now.